tlsobs-api/handlers.go

```diff
@@ -636,6 +638,45 @@ func PreflightHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("preflighted"))
 }
 
+func heartbeatHandler(w http.ResponseWriter, r *http.Request) {
+	val := r.Context().Value(ctxDBKey)
+	if val == nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	db, ok := val.(*pg.DB)
+	if !ok || db.Ping() != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
```

Comment:
can you add a small response body too? like `alive`. it's nicer for manual testing